app/MindWork AI Studio/Dialogs/ChatTemplateDialog.razor

```diff
@@ -0,0 +1,128 @@
+                Variant="Variant.Outlined"
+                UserAttributes="@SPELLCHECK_ATTRIBUTES"
+            />
+
+            <MudTextField
+                T="string"
+                @bind-Text="@this.DataSystemPrompt"
+                Validation="@this.ValidateSystemPrompt"
+                AdornmentIcon="@Icons.Material.Filled.ListAlt"
+                Adornment="Adornment.Start"
+                Immediate="@true"
+                Label="@T("What system prompt do you want to use?")"
+                Variant="Variant.Outlined"
+                Lines="6"
+                AutoGrow="@true"
+                MaxLines="12"
+                Class="mb-3"
+                MaxLength="444"
+                UserAttributes="@SPELLCHECK_ATTRIBUTES"
+                HelperText="@T("Tell the AI your system prompt.")"
+            />
+            
+            <MudTooltip Text="@T("Are you unsure which system prompt to use? You can simply start with the default system prompt that AI Studio uses for all chats.")">
+                <MudButton Class="mb-3" Color="Color.Primary" OnClick="@this.UseDefaultSystemPrompt" Size="Size.Small" StartIcon="@Icons.Material.Filled.ListAlt" Variant="Variant.Filled">@T("Use the default system prompt")</MudButton>
+            </MudTooltip>
+            
+            <MudTooltip Text="@T("Using some chat templates in tandem with profiles might cause issues. Therefore, you can preliminarily block the usage of profiles here.")">
+                <MudSwitch @bind-Value="allowProfileUsage" Class="mb-3" Color="Color.Primary" Label="@T("Allow the use of profiles together with this chat template?")" ThumbIcon="@Icons.Material.Filled.Person4" ThumbIconColor="Color.Default" />
+            </MudTooltip>
+            
+            <MudText Typo="Typo.h6" Class="mb-3">
+                @T("Example Conversation")
+            </MudText>
+            
+            <MudJustifiedText Typo="Typo.body1" Class="mb-3">
+                @T("Add messages of an example conversation (user prompt followed by assistant prompt) to demonstrate the desired interaction pattern. These examples help the AI understand your expectations by showing it the correct format, style, and content of responses before it receives actual user inputs.")
+            </MudJustifiedText>
+            
+            <MudTable CanCancelEdit="true" Class="mt-3 mb-6" CommitEditTooltip="@T("Commit Changes")" Elevation="10" FixedHeader="true" Items="@AdditionalMessages" Outlined="true" RowEditCancel="@this.ResetItemToOriginalValues" RowEditCommit="@this.ItemHasBeenCommitted" RowEditPreview="@this.BackupItem">
+                <ColGroup>
+                    <col style="width: 16em;" />
+                    <col/>
+                    <col style="width: 16em;" />
+                </ColGroup>
+                <HeaderContent>
+                    <MudTh>Role</MudTh>
+                    <MudTh>Entry</MudTh>
+                    <MudTh Style="text-align:center">Actions</MudTh>
```

Comment:
Please add the column headers to the I18N system so we can translate them into other languages.